daemon/cmd/daemon.go

```diff
@@ -1129,13 +1135,9 @@ func (d *Daemon) ReloadOnDeviceChange(devices []string) {
 	}
 
 	if option.Config.EnableNodePort {
-		if err := node.InitNodePortAddrs(devices, option.Config.LBDevInheritIPAddr); err != nil {
```

Comment:
`LBDevInheritIPAddr` is no longer referenced. Does this change ignore  `bpf-lb-dev-ip-addr-inherit` flag users specified?

Reply:
Will fix this in follow-up PR once we've figured out the right way to do it.